app/assets/javascripts/fae/form/_filtering.js

```diff
@@ -179,7 +179,10 @@ Fae.form.filtering = {
   _addSortingListeners: function() {
     var _this = this;
 
-    $('th[data-sort]').on('click', function() {
+    // Fizzle if listeneres have already been attached
+    if (this._are_sorting_listeners_setup === true) { return; }
+
+    $('body').on('click', 'th[data-sort]', function() {
```

Comment:
Continuing to use `$('body')` until issue of `FCH.$body` returning `[]` is resolved